app/templates/admin/administer-user-modal.coco.pug

```diff
@@ -24,6 +24,8 @@ block modal-header-content
           a(href='/outcomes-report/' + outcomesReportKind + '/' + view.user.get('_id') data-i18n="outcomes.view_outcomes_report")
 
     .col-sm-3
+      if me.isAdmin()
+        button#user-spy-btn.btn.btn-primary.btn-block Spy
```

Comment:
great feature, we should advertise to the team internally though about this!!! 🥳

Reply:
I think they'll see it no problem!